src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/extensions */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = new Big(numberOne);
+  const y = new Big(numberTwo);
+
+  if (operation === '-') {
+    return x.minus(y);
+  }
+
+  if (operation === '+') {
+    return x.plus(y);
+  }
+
+  if (operation === 'X') {
+    return x.mul(y);
+  }
```

Comment:
Normally, multiplication should be the 'star' ( * ) right ? hahaha

Reply:
Yeah, that would be correct, under normal circumstances, haha